src/tests/api-tests.ts

```diff
@@ -4699,6 +4699,45 @@ describe('api tests', () => {
           execution_cost_write_count: 0,
           execution_cost_write_length: 0,
         },
+        {
+          tx_id: '0x1234',
+          tx_status: 'success',
+          tx_result: {
+            hex: '0x0100000000000000000000000000000001', // u1
+            repr: 'u1',
+          },
+          tx_type: 'coinbase',
+          fee_rate: '1234',
+          is_unanchored: false,
+          nonce: 0,
+          anchor_mode: 'any',
+          sender_address: 'ST3J8EVYHVKH6XXPD61EE8XEHW4Y2K83861225AB1',
+          sponsored: false,
+          post_condition_mode: 'allow',
+          post_conditions: [],
+          block_hash: '0x1234',
+          block_height: 1,
+          burn_block_time: 39486,
+          burn_block_time_iso: '1970-01-01T10:58:06.000Z',
+          canonical: true,
+          microblock_canonical: true,
+          microblock_hash: '',
+          microblock_sequence: I32_MAX,
+          parent_block_hash: '',
+          parent_burn_block_time: 1626122935,
+          parent_burn_block_time_iso: '2021-07-12T20:48:55.000Z',
+          tx_index: 4,
+          coinbase_payload: {
+            data: '0x636f696e62617365206869',
+          },
+          event_count: 5,
+          events: [],
+          execution_cost_read_count: 0,
+          execution_cost_read_length: 0,
+          execution_cost_runtime: 0,
+          execution_cost_write_count: 0,
+          execution_cost_write_length: 0,
+        },
         {
```

Comment:
Should we include the anchored unit tests from PR https://github.com/hirosystems/stacks-blockchain-api/pull/946 if we go ahead with merging this one and not the other?

Reply:
Yep, absolutely. I'll add more tests and get back to you.